modules/@apostrophecms/schema/ui/apos/logic/AposInputSlug.js

```diff
@@ -48,6 +49,9 @@ export default {
     },
     localePrefix() {
       return this.field.page && apos.i18n.locales[apos.i18n.locale].prefix;
+    },
+    stripAccents() {
+      return apos.i18n.stripUrlAccents === true;
```

Comment:
Having this here will make it difficult to have this feature enabled/disabled per locales.

Reply:
As discussed the feature is not per locale.